### ISSUE TITLE
Requires Ruby and Sass??

### ISSUE THREAD

**Issue by kanethal** [meanjs/generator-meanjs#202]
Received the following error after running the generator (tried version 4.2 and master)
```
Running "env:dev" (env) task

Running "sass:dist" (sass) task
Warning:
You need to have Ruby and Sass installed and in your PATH for this task to work.
More info: https://github.com/gruntjs/grunt-contrib-sass
 Use --force to continue.
```
Ruby and Sass are not listed as requirements in the docs at http://meanjs.org/docs/0.4.x/
Are these requirements then? Or maybe I've done something horribly wrong... :(

**Comment by ilanbiala** [comment]
Yes they are requirements. Can you make a PR so others are aware?

**Comment by kanethal** [comment]
I would love to. But I'm a freaking newb and wouldn't know what to do...
I'll try to educate myself on that later if I get some time

**Comment by rof20004** [comment]
Try this:

$ sudo apt-get install ruby (if you have no one yet)
$ sudo apt-get install ruby-compass
$ sudo npm install grunt-compass --save-dev

**Comment by kanethal** [comment]
Thanks but I'm working in powershell

**Comment by rof20004** [comment]
Ok, then install ruby and execute?

gem install compass

**Comment by kanethal** [comment]
Right, I had Ruby once upon a time before I became interested in node.
Trying to decide if I want to go with this solution

**Comment by stephengtuggy** [comment]
@rof20004 your instructions worked for me. Thanks.

I have Windows 10 (x64) as my main OS. I installed Ruby 2.2.4 x64 from [http://rubyinstaller.org/downloads/](http://rubyinstaller.org/downloads/). I used Git Bash as my command line environment (installed with GitHub Desktop for Windows). I opened Git Bash and ran "gem install compass". I was then able to run grunt with no issues, and view the boilerplate app in Firefox at localhost:3000 .

Note that I had some issues with yo in Git Bash. When I tried to use the arrow keys to choose which version of meanjs to generate, the visual indicator didn't move. I had to just press the down arrow twice and then press Enter, and trust that it chose meanjs version 0.4.2 correctly. (It did, from what I can tell.) Note also that before I did THAT, I had to install Node.js version 4.4.0, x64, from nodejs.org, and then run npm install -g npm to update to the latest version of npm. Otherwise I ran into all sorts of issues and warning messages.

I'm also trying the same thing in an Ubuntu 15.10 x64 VM under VirtualBox 5.0.16. I used your sudo apt-get, etc. commands to get going. It seems to be working, for the most part; I just need to install MongoDB still.

**Comment by stephengtuggy** [comment]
It turns out that the latest version of MongoDB, from mongodb.org, is not fully supported on Ubuntu 15.10, only on LTS versions of Ubuntu (such as 14.04). So I ended up setting up an Ubuntu 14.04 LTS x64 VM to run my MEAN application. Got it working!
